Userland/Libraries/LibWeb/IndexedDB/IDBRequest.cpp

```diff
@@ -27,6 +27,12 @@ void IDBRequest::initialize(JS::Realm& realm)
     WEB_SET_PROTOTYPE_FOR_INTERFACE(IDBRequest);
 }
 
+void IDBRequest::visit_edges(Visitor& visitor)
+{
+    Base::visit_edges(visitor);
+    visitor.visit(m_error);
```

Comment:
m_result should be visited as well incase it ever ends up storing an Object which is allocated on the heap